### DISCUSSION TITLE
Just a question about naming

### DISCUSSION THREAD

**Discussion by RafaRioFal4699** [xerial/sqlite-jdbc#750] [Q&A] [answered]
I'm following most of issues opened in this reporitory because I'm waiting the next version of the JDBC. But I have a question: when you mention for example version 3.39.0, are you referring to SQLite's version or JDBC's version?

**Comment by gotson** [comment, marked as answer]
SQLite version. The version of `sqlite-jdbc` follows the SQLite version, since it is packaged within the jar. We will soon release `sqlite-jdbc` `3.39.2`, which will include SQLite `3.39.2`.
